panda/src/x11display/x11GraphicsWindow.cxx

```diff
@@ -2051,7 +2051,7 @@ get_keyboard_map() const {
       continue;
     }
 
-    KeySym sym = XKeycodeToKeysym(_display, k, 0);
+    KeySym sym = XkbKeycodeToKeysym(_display, k, 0, 0);
```

Comment:
It is part of the X11, Xkb is just an extension ... it may not present on very old X servers.